### PR TITLE
Readme Updated

### PR DESCRIPTION
**Pull request by eduardohoraciosanto**
From Develop, Updated Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---EN---
 This is a Simple Console App to test with my friend the usage of GitFlow.
-
+The correct way of adding stuff is branching from develop.
 ---ESP---
```
Comment (BpointCat) [inline README.md]: You are missing an Enter. Please add a blank Line.

Te faltó el enter, por favor agregarlo para mayor legibilidad.

Comment (eduardohoraciosanto) [reply README.md]: Are you Sure an enter is needed here?

Estás seguro que hace falta Enter acá?

Comment (BpointCat) [reply README.md]: yes.

Comment (eduardohoraciosanto) [reply README.md]: Ok @BpointCat, Change done!

**Pull request merged**